web/core/store/label.store.ts

```
@@ -92,45 +90,45 @@ export class LabelStore implements ILabelStore {
       "sort_order"
     );
   }
 
   /**
    * Returns the labelMap in a tree format
    */
-  get projectLabelsTree() {
-    if (!this.projectLabels) return;
-    return buildTree(this.projectLabels);
+  get labelsTree() {
+    if (!this.labels) return;
```

Biome (1.9.4)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)